project/project.go

```diff
@@ -15,6 +15,9 @@ import (
 	"github.com/docker/libcompose/yaml"
 )
 
+// ComposeVersion is name of docker-compose.yml file syntax supported version
+const ComposeVersion = "1.5.0"
```

Comment:
If we switch this to "2", does that just indicate that we have support for v2 files? If so, I think it shouldn't hurt to bump this now.